src/components/Fieldset/Fieldset.module.css

```diff
@@ -5,18 +5,28 @@
 \*------------------------------------*/
 
 /**
- * 1) Grouping of form fields with a legend
+ * The fieldset groups a legend and several controls.
+ * 1) Overrides default browser styling.
  */
 .fieldset {
-  @mixin fieldsetStyles;
```

Comment:
```
@define-mixin fieldsetStyles {
  display: block;
  margin: 0;
  padding: 0;
  border: none;
}

@define-mixin legendReset {
  display: inline-block;
  padding: 0;
  border: none;
}
```
Think I could be using these instead but default display for `fieldset` is block anyways.
As for legend, default display is also `block` but for this case don't think it would matter?